lib/signable/template.rb

```diff
@@ -17,5 +17,9 @@ def update
     def delete
       raise "not available"
     end
+
+    def remind
+      raise "not available"
```

Comment:
```suggestion
      raise 'not available'
```
Could we replace these double quotes for single quotes please?
Also for the other strings in this class please 🙂

Reply:
This is how strings are quoted throughout this gem.
We've raised a ticket to setup Rubocop in this gem repository.
I left it like this for consistency in the meantime, then when Rubocop is setup all these issues will be removed from the gem by Rubocop's auto-correct feature.

Reply:
Sounds good to me. Thanks for the explanation. 🙂